arch/Kconfig

```diff
@@ -37,6 +37,18 @@ config ARM
 	help
 	  ARM architecture
 
+config SPARC
+	bool
+	select ARCH_IS_SET
+	select USE_SWITCH
+	select USE_SWITCH_SUPPORTED
+	select BIG_ENDIAN
```

Comment:
I believe this is our first big endian arch. Are all SPARC ICs big endian, this is not configurable like with Arm?

Reply:
Yes, the 32-bit SPARC V8 architecture is always big endian.
(There is also a 64-bit SPARC architecture specification (V9), often referred to as sparc64, which is bi-endian.)